### ISSUE TITLE
ModSecurity V2 - missing Geo Countries

### ISSUE THREAD

**Issue by emphazer** [SpiderLabs/ModSecurity#2124]
**Describe the bug**

````
SecRule TX:REAL_IP "@geoLookup" chain
                SecRule GEO:COUNTRY_CODE "@pm BL BQ CW MF SS SX"
````
is not recognized by mod_security

**Logs and dumps**

Output of:
 1. DebugLogs (level 9)
 2. AuditLogs
 3. Error logs
 4. If there is a crash, the core dump file.

_Notice:_ Be carefully to not leak any confidential information.

**To Reproduce**

A **curl** command line that mimics the original request and reproduces the problem. Or a ModSecurity v3 test case.

[e.g: curl "modsec-full/ca/..\\..\\..\\..\\..\\..\\/\\etc/\\passwd" or [issue-394.json](https://github.com/SpiderLabs/ModSecurity/blob/v3/master/test/test-cases/regression/issue-394.json)]

**Expected behavior**

**Server (please complete the following information):**
 - ModSecurity version (and connector): Modsecurity v2.93
 - WebServer: Apache 2.4.34
 - OS (and distro): any (Redhat el7 in this case) 

**Rule Set (please complete the following information):**
 any with GEO:COUNTRY_CODE

**Additional context**

according to PR #2123

**Comment by zimmerle** [comment]
Closed as detailed on #2123.

**Comment by zimmerle** [comment]
Country codes: BL BQ CW MF SS SX were added to v2/master.
